doc/user/content/sql/show-sinks.md

```diff
@@ -12,14 +12,16 @@ aliases:
 
 ## Syntax
 
-{{< diagram "show-sinks.svg" >}}
+```mzsql
+SHOW SINKS [ FROM <schema_name> ] [ IN CLUSTER <cluster_name> ]
+```
 
 ## Details
 
-Field | Use
-------|-----
-_schema&lowbar;name_ | The schema to show sinks from. Defaults to first resolvable schema in the search path. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
-_cluster&lowbar;name_ | The cluster to show sinks from. If omitted, sinks from all clusters are shown. For available clusters, see [`SHOW CLUSTERS`](../show-clusters).
+Option                        | Description
+------------------------------|------------
+**FROM** <schema_name>        | Specifies the schema to show sinks from. Defaults to first resolvable schema in the search path. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
```

Comment:
```suggestion
**FROM** <schema_name>        | The schema to show sinks from. Defaults to first resolvable schema in the search path. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
```

Reply:
I was mindlessly removing the "Specifies" because...less words > more words? Will stop here just in case you have a strong preference for including it, though.

Reply:
So, I think technically, "Specifies ..." is more accurate since "The schema to show sinks from" would actually describe `<schema_name>`. Whether I have a strong preference ... eh.  Given that I know I'll do additional passes through all these files in the future, only a mild preference ... more so in that in later passes (which could even be as I work on next batch of these), it's much easier to remove text than add.

Reply:
As for the two outliers in this batch, just an oversight on my part.  Am using Cursor, and sometimes, I think I'd already accepted the suggestion and don't actually tab to accept the suggestion (whereas other times, I think I didn't accept the suggestion and accidentally just add a tab).

Reply:
Oh ... Let me try in the next patch with "Specifies" ->  "If specified, shows ..."